### DISCUSSION TITLE
[04/07/2024] Score Community Meeting

### DISCUSSION THREAD

**Discussion by sujaya-sys** [score-spec/spec#86] [Community meetings]
**Info**

| Community call | Info |
|:-----------|:------------|
| Meeting Link | Join via [Google Meet](https://meet.google.com/znt-usdc-hzs) or call +49 40 8081618260 (Pin: 599 887 196)
| Meeting Agenda & Notes | Add to our agenda or review minutes [here](https://github.com/score-spec/spec/discussions/categories/community-meetings)
| Meeting Time | 1:00-2:00pm UTC, every first Thursday of the month

**Agenda**

* Review of [open PRs](https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+user%3Ascore-spec) and  newly added [issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Ascore-spec+)

* score-humanitec & score-helm have been marked as deprecated. Let's scan open issues & PRs to understand which one's we want to move / document before closing

* Next steps CNCF sandbox admission: https://github.com/cncf/sandbox/issues/79#issuecomment-2178013391

* Update of the Score website to match latest changes of our reference implementations

* Move website to Score organisation (Hugo)

**Comment by mathieu-benoit** [comment]
2 other topics if possible:
- New App Dev WG within the TAG App Delivery: https://tag-app-delivery.cncf.io/wgs/app-development/. Does Score fits more with this new WG in addition to the Platforms one? Should we connect with them?
  - "Score is the glue between the 2".
  - Let's illustrate where Score is in this schema here: https://github.com/cncf/tag-app-delivery/blob/main/website/content/en/wgs/app-development/charter/imgs/developer-workflows.png.
  - Let's present in one of their coming community meetings and get their feedback.
- Could we have a place (GitHub or website) where we show blog posts and videos talking about Score?
  - Will be an entry on the main website.

**Comment by astromechza** [comment]
Good chat 👍 

I'll make sure I go over some missing provisioners and import mechanisms.

**Comment by sujaya-sys** [comment]
Here's a summary of what we discussed:

- Concern about both reference implementations score-compose and score-k8s offering the same level of support when it comes to resource provisioners to ensure consistency. ATM [mongodb](https://github.com/score-spec/score-k8s/issues/12) and [rabbitmq](https://github.com/score-spec/score-k8s/issues/11) are missing. 

- Related to the above point we discussed the option of a dedicated repo for custom / community created resource provisioners. Now that we have a series of provisioners available for our two reference implementations, external contributors can use these as a blueprint to build their own or replicate them in the context of their Score implementations.

- The Score website will be moved to Hugo and managed within the Score GitHub org soon. This is work in progress.

To Do's:

- [x] @chrishumanitec to scan deprecated repositories [score-helm](https://github.com/score-spec/score-helm), [score-helm-charts](https://github.com/score-spec/score-helm-charts) and [score-humanitec](https://github.com/score-spec/score-humanitec). Are there any open issues or PR's that we'd like to document before closing them off e.g. feature ideas that might be relevant for Score in general or other implementations? Once everything has been closed off we can go ahead and archive these repos. 

- [ ] @Florian to add a list of resources to the new website as pointed out by Mathieu [here](https://github.com/score-spec/spec/discussions/86#discussioncomment-9959112)

- [ ] As pointed out by Mathieu [here](https://github.com/score-spec/spec/discussions/86#discussioncomment-9959112) does Score fits into both the WG platform and WG App Delivery group. @sujaya-sys to come up with an idea on how to refine our messaging here.
